src/cpp/ext/csm/metadata_exchange.cc

```diff
@@ -70,13 +68,13 @@ constexpr absl::string_view kMetadataExchangeProjectIdKey = "project_id";
 constexpr absl::string_view kMetadataExchangeCanonicalServiceKey =
     "canonical_service";
 // The keys that will be used for the local attributes when recording metrics.
+constexpr absl::string_view kCanonicalServiceAttribute =
+    "csm.canonical_service";
```

Comment:
I think we want this to be `csm.workload_canonical_service`.

Reply:
done